profiles/pdf.md

```diff
@@ -0,0 +1,68 @@
+# PDF Profile
+
+**Editors:**
+
+* Michael Benowitz ([NYPL](https://www.nypl.org))
+* Laurent Le Meur ([EDRLab](https://www.edrlab.org))
+
+**Participate:**
+
+* [GitHub readium/webpub-manifest](https://github.com/readium/webpub-manifest)
+* [File an issue](https://github.com/readium/webpub-manifest/issues)
+
+## Example
+
+```json
+{
+  "@context": "http://readium.org/webpub-manifest/context.jsonld",
+  
+  "metadata": {
+    "@type": "http://schema.org/Book",
+    "conformsTo": "https://readium.org/webpub-manifest/profiles/pdf",
+    "title": "Moby-Dick",
+    "author": "Herman Melville",
+    "language": "en"
+  },
+
+  "links": [
+    {"rel": "self", "href": "http://example.org/manifest.json", "type": "application/webpub+json"},
+  ],
+
+  "readingOrder": [
+    {
+      "href": "http://example.org/mobydick-part1.pdf", 
+      "type": "application/pdf", 
+    },
+    {
+      "href": "http://example.org/mobydick-part2.pdf", 
+      "type": "application/pdf", 
+    }
+  ],
+  
+  "resources": [
+    {"rel": "cover", "href": "http://example.org/cover.jpeg", "type": "image/jpeg", "height": 600, "width": 300}
+  ]
+}
+```
+
+## Introduction
+
+The goal of this specification is to provide a profile of the [Readium Web Publication Manifest](https://readium.org/webpub-manifest) dedicated to pdf documents.
```

Comment:
Should be "PDF documents" instead of lowercase pdf.

Reply:
done